R/cluster_sce.R

```diff
@@ -0,0 +1,42 @@
+#' Perform clustering on a SingleCellExperiment object
+#'
+#' @param sce SingleCellExperiment object that requires clustering
+#' @param pca_name The name used to access the PCA results stored in the
+#'   SingleCellExperiment object; default is "PCA"
+#' @param BLUSPARAM A BlusterParam object specifying the clustering algorithm to
+#'   use and any additional clustering options
+#' @param cluster_column_name The name of the column to store the clustering
+#'   results in the SCE; for naming you may want to include the type of clustering
+#'   and k value of centers, e.g. `kmeans_10`
+#' @param seed Seed for reproducibility of clustering results
+#' @param ... Additional arguments to provide to `bluster::clusterRows()`
+#'
+#' @import SingleCellExperiment
+#'
+#' @return SingleCellExperiment object containing clustering results
+#'
+#' @examples
+#'   Perform K-means clustering with 10 centers
+#'   cluster_sce(sce, "PCA", bluster::KmeansParam(centers = 10), "kmeans_10")
```

Comment:
This is missing a `\dontrun{}` around it and also a `#` before the comment. 
```suggestion
#' @examples
#'  \dontrun{
#'   # Perform K-means clustering with 10 centers
#'   cluster_sce(sce, "PCA", bluster::KmeansParam(centers = 10), "kmeans_10")
#'  }
```
Also to test out checks locally I would encourage you to run `devtools::check()` while in the project if you are not doing that already. It will help catch things like these.

Reply:
good catch, @allyhawkins! I definitely second running `devtools::check()` which I should have done as part of review but did not (happily the 🤖 does though!) - protip, use `cmd+shift+e` shortcut in RStudio.